CMakeLists.txt

```diff
@@ -65,12 +65,16 @@ endif()
 # Library mode: SHARED (default) or STATIC
 #
 set(CTK_LIBRARY_MODE "SHARED")
+set(MY_LIBRARY_TYPE "SHARED")
+add_definitions(-DCTK_SHARED)
```

Comment:
Variable `CTK_LIBRARY_MODE` should be used where relevant, there is no need to re-introduce an other variable